server/sonar-web/src/main/js/app/components/nav/global/global-nav-menu.js

```diff
@@ -77,9 +77,13 @@ export default React.createClass({
   },
 
   renderAdministrationLink () {
-    if (!window.SS.isUserAdmin) {
+    // FIXME check permissions
+    if (!this.props.currentUser) {
       return null;
     }
+    // if (!window.SS.isUserAdmin) {
+    //   return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3ACommentedCode)